dev/run_full_stack.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+# Store current directory.
+pushd `pwd`
+# We have to run docker compose from this directory for it to pick up the .env file.
+cd `dirname "$0"`
```

Comment:
can be pushd `dirname "$0"` as pushd also changes directory

Reply:
oh, did not know that. updated.